src/grid/atomgrid.py

```diff
@@ -261,44 +271,38 @@ def get_shell_grid(self, index, r_sq=True):
             new_wts = wts
         return AngularGrid(pts, new_wts)
 
-    def convert_point_to_sph(self, points):
-        """Convert a set of Cartesian points to sphercial coordinates.
+    def convert_cart_to_sph(self, points=None, center=None):
+        """Convert a set of points from cartesian to spherical coordinates.
+
+        The spherical center is selected as the atom if not given
+        The points is selected as all the atomic grid points if not given
```

Comment:
Isn't it better to add these to the arguments definition in the `Parameters` section? It better reflects what is used if they are `None`.